xbmc/video/SetInfoTag.h

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+class TiXmlNode;
+class TiXmlElement;
+
+class CSetInfoTag
+{
+public:
+  CSetInfoTag() { Reset(); }
+  virtual ~CSetInfoTag() = default;
+  void Reset();
+  /* \brief Load information to a setinfotag from an XML element
+
+   \param element    the root XML element to parse.
+   \param append     whether information should be added to the existing tag, or whether it should be reset first.
+   \param prioritise if appending, whether additive tags should be prioritised (i.e. replace or prepend) over existing values. Defaults to false.
+
+   \sa ParseNative
+   */
+  bool Load(const TiXmlElement* element, bool append = false, bool prioritise = false);
+  bool IsEmpty() const;
+
+  void SetOverview(std::string overview);
+  const std::string& GetOverview() const;
+
+  void SetTitle(std::string title);
+  const std::string& GetTitle() const;
+
+  void Merge(CSetInfoTag& other);
+  void Copy(CSetInfoTag& other);
+
+  std::string m_title; // Title of the movie set
+  int m_id; // ID of movie set in database
```

Comment:
`int m_id{-1};`to be on the safe side. I know that currently not needed, because you call "Reset()" in the ctor, but still this removes one source for bugs.